public/scripts/requiredWords.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+function getRandomWords(wordCount) {
+  const randomWords = require('random-words')
```

Comment:
General practice is to put your requires at the top of the file, under the `use strict` directive. That way you can sort of cache the module instead of loading it in this file each time you make a call to this function